src/api/ledger/parse/utils.js

```diff
@@ -1,10 +1,25 @@
 /* @flow */
 'use strict';
 const _ = require('lodash');
+const BigNumber = require('bignumber.js');
 const transactionParser = require('ripple-lib-transactionparser');
 const toTimestamp = require('../../../core/utils').toTimestamp;
 const utils = require('../utils');
 
+/*:: type XRPAmount = {currency: string, value: string} */
```

Comment:
Can these definitions be placed somewhere accessible so they don't need to be repeated locally?